core/store/orm/schema.go

```diff
@@ -82,6 +81,7 @@ type ConfigSchema struct {
 	MinimumContractPayment                    assets.Link     `env:"MINIMUM_CONTRACT_PAYMENT" default:"1000000000000000000"`
 	MinimumRequestExpiration                  uint64          `env:"MINIMUM_REQUEST_EXPIRATION" default:"300"`
 	OCRObservationTimeout                     time.Duration   `env:"OCR_OBSERVATION_TIMEOUT" default:"13s"`
+	OCRObservationGracePeriod                 time.Duration   `env:"OCR_OBSERVATION_GRACE_PERIOD" default:"2s"`
```

Comment:
With these two defaults for ObservationTimeout and ObservationGracePeriod, we can handle up to to n-1 infinitely slow APIs and still return observations without setting any env vars.